app/lib/l10n/app_de.arb

```diff
@@ -41,6 +41,7 @@
   "accentColor": "Akzent Farbe",
   "standard": "Standart",
   "dynamicColor": "Dynamisch",
+  "enableSubstitutionsInfo": "Zeige Anmerkungen zu Vertretungen",
```

Comment:
This can be shorter. I think just "Anmerkungen anzeigen" is enough. Thoughts?

Reply:
If this is moved to an applet setting section, it should be sufficient. `Anmerkungen anzeigen` in the context of the general app is somewhat misleading.